cl_dll/vgui_ScorePanel.cpp

```diff
@@ -102,6 +102,10 @@ ScorePanel::ScorePanel(int x,int y,int wide,int tall) : Panel(x,y,wide,tall)
 	Font *tfont = pSchemes->getFont(hTitleScheme);
 	Font *smallfont = pSchemes->getFont(hSmallScheme);
 
+	m_UFont = UnicodeTextImage::createFont("Arial", YRES(12), 300, true);
+	m_UTitleFont = UnicodeTextImage::createFont("Arial", tfont->getTall(), 700, true);
+	m_USmallFont = UnicodeTextImage::createFont("Arial", smallfont->getTall(), 400, true);
```

Comment:
Do these need to be freed in the destructor?

Reply:
No, all loaded fonts are freed by the engine's font manager on shutdown.

Reply:
So it won't build up on disconnect and reconnect etc?

Reply:
Viewport and ScorePanel are constructed only once (first time connecting to a server) so fonts are only created once.